packages/bsky/src/api/app/bsky/feed/getLikes.ts

```diff
@@ -1,78 +1,127 @@
 import { mapDefined } from '@atproto/common'
 import { Server } from '../../../../lexicon'
+import { QueryParams } from '../../../../lexicon/types/app/bsky/feed/getLikes'
 import { paginate, TimeCidKeyset } from '../../../../db/pagination'
 import AppContext from '../../../../context'
 import { notSoftDeletedClause } from '../../../../db/util'
+import { BlockAndMuteState, GraphService } from '../../../../services/graph'
+import { ActorInfoMap, ActorService } from '../../../../services/actor'
+import { Actor } from '../../../../db/tables/actor'
+import { Database } from '../../../../db'
+import { createPipeline } from '../../../../pipeline'
 
 export default function (server: Server, ctx: AppContext) {
+  const getLikes = createPipeline(skeleton, hydration, noBlocks, presentation)
```

Comment:
looks like we were originally filtering mutes here - which tbh i think i like it better just blocks 🤔

Reply:
If that's based off of the name `filterBlocksAndMutes()` below, it's a little confusing but it was only applying blocks, since it was only being passed an implementation for `getBlockPairs()` and not `getMutePairs()`.  In any case, I'll leave it as-is 👍 👍

Reply:
ah right on - yup 👍